metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/PcapServiceImpl.java

```diff
@@ -199,6 +208,37 @@ public InputStream getRawPcap(String username, String jobId, Integer page) throw
     return inputStream;
   }
 
+  @Override
+  public Map<String, Object> getConfiguration(String username, String jobId) throws RestException {
+    Map<String, Object> configuration = new HashMap<>();
+    try {
+      Map<String, Object> jobConfiguration = jobManager.getJob(username, jobId).getConfiguration();
```

Comment:
Will `jobManager.getJob(username, jobId)` never return null?  I see null checks in other places where similar calls are made.